packages/chatbot-eval-mongodb-public/src/braintrust/evaluateRagConversationsData.ts

```diff
@@ -21,11 +21,17 @@ const runs = [
   //   model: "gpt-4o-mini",
   //   generatedDataRunId: new Object("66c35ec36e955a072a6f8784").toString(),
   // },
+  //   {
+  //     model: "gpt-35-preprocessor-gpt4o-mini-responder",
+  //     generatedDataRunId: new Object("66c37e6c97b47998dd59e87e").toString(),
+  //     description: `Uses GPT-3.5 for pre-processsing tasks: metadata extraction, user input guardrail, and query rewriting.
+  // Uses GPT-4o mini for the main responder.`,
+  //   },
   {
-    model: "gpt-35-preprocessor-gpt4o-mini-responder",
-    generatedDataRunId: new Object("66c37e6c97b47998dd59e87e").toString(),
+    model: "gpt-35-preprocessor-gpt4o-responder",
+    generatedDataRunId: new Object("66cc95022525d293a0ef4cc3").toString(),
```

Comment:
```suggestion
    generatedDataRunId: new ObjectId("66cc95022525d293a0ef4cc3").toString(),
```
I think this is an issue throughout this file - fun little quirk of JS that this works currently!

Reply:
funny that this works. will fix before merge